packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Image.tsx

```diff
@@ -10,7 +10,7 @@ import {
 } from '@sitecore-jss/sitecore-jss-nextjs';
 
 interface Fields {
-  Image: ImageField;
+  Image: ImageField & { metadata?: { [key: string]: unknown } };
```

Comment:
Do you need to add _metadata_ type since it's a part of _ImageField_?

Reply:
As I can see it's not a part pf `ImageField`. I checked here https://github.com/Sitecore/jss/blob/feature/editing-integration/packages/sitecore-jss-react/src/components/Image.tsx#L40